### PR TITLE
Rework and test

### PR DESCRIPTION
**Pull request by wistefan**

### CODE REVIEW

**Review by beknazaresenbek — APPROVED** [review]

**Review by dwendland — APPROVED** [review]

File: doc/LOCAL.MD [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+              "odrl:action": {
+                "@id": "odrl:read"
+              }
+            }
+          }'
+```
+
+Data can be created through the NGSI-LD API itself. In order to make interaction easier, its directly available through
+an ingress at ```http://scorpio-provider.127.0.0.1.nip.io/ngsi-ld/v1```. In
+real environments, no endpoint should be publicly available without beeing protected by the authorization framework.
```
Comment (pulledtim) [inline doc/LOCAL.MD:253]:
```suggestion
real environments, no endpoint should be publicly available without being protected by the authorization framework.
```

File: doc/LOCAL.MD [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+
+### Participant Identity
+
+In a Data Space, every participant requires an identity. The FIWARE Data Space relies on [Decentralized Identifiers](https://www.w3.org/TR/did-core/)
+to identify its participants. While the concrete scheme to be used for a Data Space needs to be decided by its requirements, 
+the local installation uses [did:key](https://w3c-ccg.github.io/did-method-key/). While the did's are not well readable for humans,
+they are well-supported, can be resolved without any external interaction and can easily be generated within the deployment. That makes
+them a perfect fit for the local use-case. 
+All participants(e.g. the consumer and the participant) get a did generated on installation, by using the [did-helper](https://github.com/wistefan/did-helper). 
+The identities and connected key-material is automatically distributed in the cluster and set in the componets that require it.
```
Comment (pulledtim) [inline doc/LOCAL.MD:454]:
```suggestion
The identities and connected key-material is automatically distributed in the cluster and set in the components that require it.
```

File: it/src/test/resources/policies/energyReport.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "@context": {
+    "dc": "http://purl.org/dc/elements/1.1/",
+    "dct": "http://purl.org/dc/terms/",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "odrl": "http://www.w3.org/ns/odrl/2/",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "TMF-PCMA": "https://tmf-open-api-table-documents.s3.eu-west-1.amazonaws.com/OpenApiTable/4.1.0/user_guides/TMF620_Product_Catalog_Management_API_v4.1.0_user_guides.pdf"
```
Comment (pulledtim) [inline it/src/test/resources/policies/energyReport.json:9]: I thought the content of context must be a json?

**Review by pulledtim — APPROVED** [review]

File: it/src/test/resources/policies/energyReport.json (hunk 3 above)
Comment (wistefan) [reply it/src/test/resources/policies/energyReport.json:9]: I just took this over from the examples in dome, I'm not sure about that. But we dont need the tmf part anyways, I'll remove it.

**Pull request merged**